src/main/webapp/event-info-page/script.js

```diff
@@ -1,11 +1,56 @@
-function createMap() {
+// globals = {
+//     event_id: 0,
+// }
+
+// window.onload = function() {
+//     this.getEventId();
+//     //this.getHeader();
+//     //this.loadEvents();
+//     //this.document.getElementById('add-event').setAttribute('onclick', `location.href = '../add_event_page/add_event.html?school_id=${globals.school_id}'`);
+// }
+
+function createMap(_lat, _lng) {
     const map = new google.maps.Map(
         document.getElementById('map'),
-        {center: {lat: 33.97919947216425, lng: -117.32849493862577}, zoom: 16});
+        {center: {lat: _lat, lng: _lng}, zoom: 16});
 
     const trexMarker = new google.maps.Marker({
-      position: {lat: 33.97919947216425, lng: -117.32849493862577},
-      map: map,
-      title: 'Cycle Bun'
+      position: {lat: _lat, lng: _lng},
+      map: map
     });
-} 
\ No newline at end of file
+}
+
+function getEventId() {
+    const urlParams = new URL(location.href).searchParams;
+    const event_id = urlParams.get('event_id');
+    return event_id;
+}
+
+async function loadTasks() {
+    const event_info = await fetch(`/event?event_id=${getEventId()}`, {method: 'GET'}).then(response => response.json())
+    document.getElementById("name").innerHTML = event_info.name;
+    document.getElementById("description").innerHTML = event_info.description;
+    document.getElementById("location").innerHTML = event_info.location_name;
+    document.getElementById("date").innerHTML = event_info.date;
```

Comment:
You can convert the date string to a Date object and then adjust the formatting to your liking (see directory-page/script.js).
`date = new Date(event.date);
formatted_date = date.toLocaleString('en-US', {
    month: "short",
    day: "numeric",
    year: "numeric",
    hour: "numeric", 
    minute: "2-digit"
})`

Reply:
Thanks for the input. It looks like the better format than I wrote. I will reflect your code in my code.